Library/Homebrew/dev-cmd/pr-pull.rb

```diff
@@ -291,34 +302,49 @@ def cherry_pick_pr!(user, repo, pr, args:, path: ".")
     Utils::Git.cherry_pick!(path, "--ff", "--allow-empty", *commits, verbose: args.verbose?, resolve: args.resolve?)
   end
 
-  def formulae_need_bottles?(tap, original_commit, user, repo, pr, args:)
+  def packages_need_bottles?(tap, original_commit, user, repo, pr, args:)
```

Comment:
Casks never need bottles so I'd suggest leaving this with the same name but calling it conditionally.